sky/skylet/constants.py

```diff
@@ -89,11 +89,11 @@
 # cluster yaml is updated.
 #
 # TODO(zongheng,zhanghao): make the upgrading of skylet automatic?
-SKYLET_VERSION = '13'
+SKYLET_VERSION = '14'
 # The version of the lib files that skylet/jobs use. Whenever there is an API
 # change for the job_lib or log_lib, we need to bump this version, so that the
 # user can be notified to update their SkyPilot version on the remote cluster.
-SKYLET_LIB_VERSION = 3
+SKYLET_LIB_VERSION = 4
```

Comment:
I think if we're updating the `SKYLET_VERSION`, we don't need to also update this.